modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -14,11 +14,24 @@
 namespace cv {
 namespace gapi {
 
+/// Enum class specifies output format of stereo() function
```

Comment:
Why are there three slashes?

Reply:
Its a decision which I found 
What should I use instead?

Reply:
`///` -> `/** ...  */`